### DISCUSSION TITLE
Alternative pressure sensor MPXV7007DP

### DISCUSSION THREAD

**Discussion by gerdb** [cardonabits/haxo-hw#33] [Ideas]
The MPXV7007**DP** version of the sensor was out of stock. I've ordered a MPXV7007**GP**
Seem that this sensor is compatible cause the pressure output P2 is not used in this project. 
Datasheet: [MPXV7007-3139005.pdf](https://www.mouser.de/datasheet/2/302/MPXV7007-3139005.pdf)

**Comment by jcard0na** [comment]
Yes, GP should work fine.  And even MPXV7007GC6U, which at some point NXP was shipping free samples.
Let me know if you encounter any issues.

Cheers,

**Comment by jcard0na** [comment]
Hi @gerdb 

Can you confirm that the GP sensor worked for you?  Thanks!

**Comment by gerdb** [comment]
I have no 3D parts yet, but with the silicon tube it seems to work. I can play and I also can access the command mode with negative pressure.

**Reply by jcard0na** [reply]
Thanks for confirming!  Regarding 3D parts, I've been using https://impressao3dportugal.pt with very good results and reasonable prices.  I don't know if they ship to Germany, but might be worth asking them at info@impressao3dportugal.pt

**Reply by gerdb** [reply]
I will get my 3D parts from http://my_brother_is_going_to_make_them_for_me_as_a_birthday_present.de :birthday: :smile:
